src/components/form/form.style.js

```diff
@@ -9,7 +9,7 @@ import baseTheme from '../../style/themes/base';
 import OptionsHelper from '../../utils/helpers/options-helper';
 
 export const StyledForm = styled.form`
-  & ${StyledFormField}, ${StyledFieldset} {
+  & ${StyledFormField}, ${StyledFieldset}, ${StyledButton} {
```

Comment:
Can we find a way so users do not have to do `leftSideButtons={ <Button mb={0}` I don't think this is part of this work and we could move it to another ticket if we can't find a solution.